README.md

```diff
@@ -130,9 +223,15 @@ The code that has been specially created or modified for this branch is stored i
 - extracted: This code is literally the same code base used in the normal OpenSmalltalk-VM build.
 - plugins: The code of the different plugins developed for headless mode.
 
-## Editing the VM code in your image
-
-You can load the Pharo code of the VM using Pharo's git client Iceberg.
-You can do so by cloning directly this repository from Iceberg, or by adding an already existing clone to it.
-
-Alternatively, if you're building the VM using the instructions above, the build process does already generate a Pharo image with the VM code loaded. You'll find such Image in the `build/vmmaker` directory inside your build directory.
+### Glossary used by PharoVM
+
+- CoInterpreter: a subclass of the interpreter that know how to use (and communicate with) Cogit for the JIT service.
+- Cog: codename of the smalltalkvm developed at ??? by ???
+- Cogit: the just in time compiler used by PharoVM. See `Cogit` in the `VMMaker` package.
+- FFI (foreign function interface) allow PharoVM to call dynamically loaded C libraries.
+- headless: another codename of a previous version of PharoVM where the graphical components where removed from the VM and moved as a image responsibility (with the help of FFI).
```

Comment:
I wonder if this one is really necessary...

Reply:
Unfortunately, `headless` still appears in some parts of doc and code (and branch names) and confused me. I'm not even sure what is the current status of the thing (is this fully done and remaining related things should be removed?)

For instance, there in the README the section about directories talks about headless a lot. There is an option `--headless` on pharovm binaries. And cmake had a ALWAYS_INTERACTIVE variable that seems related.